cfme/tests/intelligence/test_chargeback_assignments.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+import cfme.fixtures.pytest_selenium as sel
+import cfme.intelligence.chargeback as cb
+import cfme.web_ui.flash as flash
+import pytest
+import random
+
+from utils.providers import setup_a_provider
+from utils import version
+
+
+@pytest.fixture(scope="module")
+def provider():
+    return setup_a_provider(prov_class="infra", prov_type="virtualcenter")
+
+
+def test_assign_compute_enterprise(provider):
```

Comment:
Actually, I think this test might not use any provider at all. Therefore if this was true, you could remove the `provider` fixture and mark the test with `@pytest.mark.sauce` so the test can run in Sauce runs :).